platform/machine/qemu/flight.go

```diff
@@ -70,37 +65,6 @@ func NewFlight(opts *Options) (platform.Flight, error) {
 		diskImagePath: opts.DiskImage,
 	}
 
-	if opts.Distribution != "cl" {
-		// don't apply CL-specific mangling
-		opts.UseVanillaImage = true
-	}
-	if !opts.UseVanillaImage {
-		info, err := util.GetImageInfo(opts.DiskImage)
-		if err != nil {
-			qf.Destroy()
-			return nil, err
-		}
-		if info.Format != "raw" {
-			// platform.MakeCLDiskTemplate() needs to be able to mount
-			// partitions
-			plog.Debug("disk image is in qcow format; not enabling console logging")
-			opts.UseVanillaImage = true
-		}
-	}
-	if !opts.UseVanillaImage {
-		plog.Debug("enabling console logging in base disk")
-		qf.diskImageFile, err = platform.MakeCLDiskTemplate(opts.DiskImage)
-		if err != nil {
-			qf.Destroy()
-			return nil, err
-		}
-		// The template file has already been deleted, ensuring that
-		// it will be cleaned up on exit.  Use a path to it that
-		// will remain stable for the lifetime of the flight without
-		// extra effort to pass FDs to subprocesses.
-		qf.diskImagePath = fmt.Sprintf("/proc/%d/fd/%d", os.Getpid(), qf.diskImageFile.Fd())
```

Comment:
This means the `diskImagePath` member can be dropped in favor of direct access to `opts.DiskImage`.  Same in `unprivqemu`.